### DISCUSSION TITLE
Problem with cloudsmith repository

### DISCUSSION THREAD

**Discussion by Cyrceja** [rabbitmq/rabbitmq-server#9828] [Other] [answered]
Hello,
I've problem with cloudsmith: 
![obraz](https://github.com/rabbitmq/rabbitmq-server/assets/149373898/178617af-a651-44f1-be86-a31583e466e7)

Do you know anything about this?

**Comment by michaelklishin** [comment]
@Cyrceja we cannot suggest anything without knowing what exactly you did. We do not guess in this community.

If you try to navigate to `ppa1.novemberain.com` and similar Cloudsmith mirrors, they do not have landing pages, and indeed will serve you a 404 page. All they are set up to do is to serve `apt` (`dnf,` `yum` in case of `yum*` subdomains) package repository requests.

**Comment by Cyrceja** [comment]
It's this same:
![obraz](https://github.com/rabbitmq/rabbitmq-server/assets/149373898/e146654e-fdb6-4f06-b836-a8d742b4bdfa)

Hi,
I'm trying to set up a repository for rhel 9, but I'm getting 404s everywhere and can't download the packages.

**Reply by michaelklishin** [reply]
Like I said, you are not supposed to access that subdomain in a Web browser. See [this doc section](https://rabbitmq.com/install-debian.html#apt-quick-start-cloudsmith) to find out how the mirror should be used with `apt`.

**Reply by Cyrceja** [reply]
I using RHEL = yum repository. 
When I try to add repositories do yum, i've an error, so i checked repos in browser and this problem I see.

**Reply by Cyrceja** [reply]
End, I try to create mirror repository at my rhel satellite and problem is too.

**Reply by michaelklishin** [reply]
If you are using `yum` then you are supposed to use [`yum2.novemberain.com`, etc](https://rabbitmq.com/install-rpm.html#cloudsmith). `ppa*` host Debian repositories.

**Comment by Cyrceja** [comment]
> It's this same: ![obraz](https://user-images.githubusercontent.com/149373898/279074173-e146654e-fdb6-4f06-b836-a8d742b4bdfa.png)
> 
> Hi, I'm trying to set up a repository for rhel 9, but I'm getting 404s everywhere and can't download the packages.


As I showed in this message, I cannot access this address either.

**Reply by michaelklishin** [reply, marked as answer]
You are **not supposed** to access it in a Web browser. That will not work, by design. You must set up a repository file for tools such as `dnf`, `yum`, or their respective mirroring solutions in order to use the mirrors. Both [Debian](https://rabbitmq.com/install-debian.html#apt-cloudsmith) and [RPM guides](https://rabbitmq.com/install-rpm.html#cloudsmith) explain how to do that.

Here is [how to set up an apt mirror](https://gist.github.com/michaelklishin/473390eaf012e2edeb72936f183192a7). The process is similar for `dnf`/`yum` repositories but uses a tool called [reposync](https://jsmith.fedorapeople.org/drafts/SMG/html/Software_Management_Guide/ch08s04.html).
